Serialization: Changed write order for extrude, tessellate, altitudeModeGroup

blipmusic opened the pull request:
Changed write order on serialization for `extrude`, `tessellate` for `write_point()` and `extrude`, `tessellate`, `altitudeMode` for `write_geom_props` so that the result validates against http://schemas.opengis.net/kml/2.2.0/ogckml22.xsd

pjsier commented:
This is just an issue of the tests now being out of date with the representation. You can update `test_write_point` and `test_write_polygon` to put the correct attributes first so that this passes. Also feel free to add the change for #23 here

blipmusic commented:
Ok, I changed the order for custom elements/children in write_placemark and tried updating the Point+Polygon tests as well. 

However, the tests still fail since `write_polygon` now also needs write order changed. What I've changed the string representation to for `test_write_point` and `test_write_polygon` validates against http://schemas.opengis.net/kml/2.2.0/ogckml22.xsd so far.

I can look into `write_polygon` this as well if you'd like, but should that be in another PR or this one?

blipmusic commented:
I've changed write order in `write_polygon` and the formatting issue in `test_write_point`. Seems to pass tests now.

pjsier commented:
Thanks for making those updates! I think this should be good, but I'll try to take a closer look later today
